src/views/Home.vue

```diff
@@ -30,16 +32,35 @@
 </template>
 
 <script>
+  import Observer from "@/components/common/Observer"
+
   export default {
     data: () => ({
-        loading: true,
-        signUpMurrens: []
+      loading: true,
+      showSignUpMurrens: [],
+      pageNumber: 1,
+      allSignUpMurrens: ''
+
     }),
     async mounted() {
-      const result = await this.$store.dispatch('fetchMurrens')
-      this.signUpMurrens = result ? result.reverse() : []
+      const result = await this.$store.dispatch('fetchMurrens', this.pageNumber)
+      this.showSignUpMurrens = result ? result.results : []
+      this.allSignUpMurrens = Math.ceil(result.count / result.results.length)
+      this.pageNumber++
       this.loading = false
     },
+    methods: {
+      async murrDefOnIntersected() {
+        if (this.pageNumber < this.allSignUpMurrens) {
```

Comment:
One more page remains, the condition needs to be corrected from `<` to `<=`

Reply:
done